### PR TITLE
fix NullPointerException within FluentDriver.buildUrl(String)

### PR DESCRIPTION
**Pull request by qoomon**

### CODE REVIEW

**Comment by filipcynarski** [comment]
@qoomon could you please extend unit tests as well?

**Comment by qoomon** [comment]
Can't Find a FluentDriverTest class where to find the matching test class

**Comment by filipcynarski** [comment]
No probs, I'll handle that

**Comment by qoomon** [comment]
Thx

File: fluentlenium-core/src/main/java/org/fluentlenium/core/FluentDriver.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ public EventsRegistry events() {
         if (events == null) {
             throw new IllegalStateException("An EventFiringWebDriver instance is required to use events. "
                     + "You should set 'eventsEnabled' configuration property to 'true' "
-                    + "or override newWebDriver() to build an EventFiringWebDriver.");
+                    + "or override newWebDriver() to FluentDriver an EventFiringWebDriver.");
```
Comment (Toilal) [inline fluentlenium-core/src/main/java/org/fluentlenium/core/FluentDriver.java]: This message should not change. Is it a mistake ?

**Comment by Toilal** [comment]
Closing in favor of #429. 

I have cherry picked your commit in the pull request, but finally this code is removed in favor of JDK URI resolve API.

**Pull request closed without merging**